COPY/usr/bin/manageiq-db-ready

```diff
@@ -25,8 +27,11 @@ end
 def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
```

Comment:
`ManageIQ::Password.try_decrypt(nil) => nil` so :+1: (just a note to self since initially looked like this might fail but it is fine)